src/imgtools/cli/autopipeline.py

```
@@ -0,0 +1,265 @@
+from dis import show_code
```

Ruff (0.8.2)

`dis.show_code` imported but unused

Remove unused import: `dis.show_code`

(F401)